packages/cli/src/ui/commands/types.ts

```diff
@@ -109,8 +109,10 @@ export type SlashCommandActionReturn =
 // The standardized contract for any command in the system.
 export interface SlashCommand {
   name: string;
-  altName?: string;
-  description?: string;
+  altNames?: string[];
+  description: string;
+
+  kind: 'built-in' | 'file';
```

Comment:
Lets create an enum for these kinds so they aren't magic strings. Otherwise in non-completion based IDEs or experiences it's harder to know what's available + AI systems do better with non-string arguments (otherwise they encourage hallucinations)